src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/move_fc_reshape_to_weights.cpp

```diff
@@ -73,7 +73,8 @@ ov::intel_cpu::MoveFCReshapeToWeights::MoveFCReshapeToWeights() {
                 return false;
 
             const auto comparison_start_pos = expected_shape.size() - node_shape.size();
-            return std::equal(node_shape.begin(), node_shape.end(), expected_shape.begin() + comparison_start_pos);
+            return std::equal(node_shape.begin(), node_shape.end(), expected_shape.begin() + comparison_start_pos) ||
+                   std::all_of(node_shape.cbegin(), node_shape.cend(), [](int dim) { return dim == 1; });
```

Comment:
Can we cover the updated condition with [unit tests](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/unit/transformations/move_fc_reshape_to_weights.cpp)?

Reply:
Agree, covered